R/zzz.R

```diff
@@ -0,0 +1,22 @@
+.onLoad <- function(libname, pkgname) {
+  # adapted from https://r-pkgs.org/code.html#sec-code-onLoad-onAttach
+  op <- options()
+  default_global_knitr <- list(teal.reporter.global_knitr = list(
+    echo = TRUE,
+    tidy.opts = list(width.cutoff = 60),
+    tidy = FALSE
```

Comment:
@kartikeyakirar I think this should be `tidy = TRUE` and it will just not work if formattR is missing. do not hide stuff like that from the user https://github.com/insightsengineering/teal.reporter/pull/218/files#diff-c8b290bb66d11d029d9e7a7585d6aeb3628e578143c2fa1a44e4661636f26f7aR37-R39

Reply:
Initially, I set it to TRUE. However, if tidy = TRUE is  set, the report does not download because the knitr process is halted due to a warning in app.

Reply:
<img width="904" alt="image" src="https://github.com/insightsengineering/teal.reporter/assets/6700955/ab0869ae-8a7a-426b-b82d-5691061da052">

Reply:
@kartikeyakirar can you make it

```r
list(teal.reporter.global_knitr = list(
    echo = TRUE,
    tidy.opts = list(width.cutoff = 60),
    tidy = requireNamespace('formattR')
```

then?

So it's visible, and exposed to the end user and not hidden